ShowSQLTables/Program.cs

```diff
@@ -9,31 +9,19 @@ static void Main(string[] args)
         {
             string connectionString = @"Data Source=.\SQLEXPRESS01;Initial Catalog=Demo1;Integrated Security=True";
 
-            string sqlExpression = "SELECT * FROM Author";
             using (SqlConnection connection = new SqlConnection(connectionString))
             {
                 connection.Open();
-                SqlCommand command = new SqlCommand(sqlExpression, connection);
-                SqlDataReader reader = command.ExecuteReader();
 
-                if (reader.HasRows)
-                {
-                    Console.WriteLine("{0} {1} {2} {3} {4}", reader.GetName(0), reader.GetName(1), reader.GetName(2), reader.GetName(3), reader.GetName(4));
+                SqlCommand command = new SqlCommand("SELECT * FROM Author", connection);
```

Comment:
`SqlCommand` and `SqlDataReader` are disposable and they are allocating unmanaged resources to connect  the database. Please include also `IDisposable` implementation here.